nifi-mock/pom.xml

```diff
@@ -58,6 +58,11 @@
             <artifactId>nifi-data-provenance-utils</artifactId>
             <version>1.13.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-kerberos-credentials-service-api</artifactId>
+            <version>1.13.0-SNAPSHOT</version>
+        </dependency>
```

Comment:
I lean towards `nifi-mock `only really providing mocks for things in `nifi-api `and `nifi-framework-api`, but not really for extensions related things like a controller service api. 

I can see the need to have shared mock impls for controller services though, maybe a new test utils module under `nifi-nar-bundles/nifi-extension-utils`?

Reply:
That makes sense, will move this into a new module.